extras/checklist.rst

```diff
@@ -0,0 +1,54 @@
+Checklist
+=========
+
+General
+-------
+
+Use a checklist in tickets to keep track of the tasks to be completed. You can
+find the checklist feature in the right sidebar in the "Checklist" tab (unless
+your admin has deactivated it). Please note that you can only add or edit a
+checklist, if you have the permission to edit the ticket.
+
+.. figure:: /images/extras/checklist/checklist-sidebar-highlight.png
+   :alt: Screenshot showing right sidebar with highlighted "Checklist" tab
+
+   Screenshot of ticket view with highlighted checklist sidebar.
+
+Usage
+-----
+
+If you want to add a checklist to a ticket, go to the "Checklist" tab and click
+either on "Add empty checklist" (1) or "Add from template" (2) after selecting
+a template. If you can't see the the "Add from template" area, there is no
+template available.
+
+.. figure:: /images/extras/checklist/checklist-creation.png
+   :alt: Checklist creation dialog
+   :align: center
+   :scale: 50%
+
+   Screenshot of adding template or new checklist.
+
+Depending on your choice, you can create your own new checklist or use a
+predefined template. In both cases, you can:
+
+- Edit the checklist title
+- Edit the checklist items (reorder, rename, add, delete)
+- Delete the complete checklist
+- Re-add a checklist, if there is no existing one
```

Comment:
Not sure what this means exactly? Do you perhaps mean remove the complete checklist? Because that action does exist in the sidebar dropdown menu.

Reply:
Not sure what you mean exactly either ;-) Changed it now to:

```
- Edit the checklist title
- Edit the checklist items (reorder, rename, add, delete)
- Remove the complete checklist
- Add a checklist, if there is no existing one
```
If there's still something unclear/wrong, let me know.